docs/api/process.md

```diff
@@ -10,6 +10,8 @@ upstream node:
 * `process.resourcesPath` String - Path to JavaScript source code.
 * `process.mas` Boolean - For Mac App Store build, this value is `true`, for
   other builds it is `undefined`.
+* `process.windowsstore` Boolean - If the app is running as a Windows Store app (appx), this value is `true`, for
```

Comment:
Shouldn't this be camelCase (`windowsStore`) like `resourcesPath` for consistency?